src/internal/connector/graph_connector_disconnected_test.go

```diff
@@ -256,11 +262,29 @@ func (suite *DisconnectedGraphConnectorSuite) TestVerifyBackupInputs() {
 				return sel.Selector
 			},
 		},
+		{
+			name:       "valid sites",
+			checkError: assert.NoError,
+			getSelector: func(t *testing.T) selectors.Selector {
+				sel := selectors.NewSharePointBackup()
+				sel.Include(sel.Sites([]string{"abc.site.foo", "bar.site.baz"}))
+				return sel.Selector
+			},
+		},
+		{
+			name:       "invalid sites",
+			checkError: assert.Error,
+			getSelector: func(t *testing.T) selectors.Selector {
+				sel := selectors.NewSharePointBackup()
+				sel.Include(sel.Sites([]string{"fnords.smarfs.brawnhilda"}))
+				return sel.Selector
+			},
+		},
```

Comment:
I'm a bit split on whether we should have some sort of test for filters as well. Now that `verifyBackupInputs` is comparing against them it seems like we should test that at least a bit

Reply:
👍 no harm in it, unit tests are easy.